resources/groundwork/web.md

```diff
@@ -78,6 +78,12 @@ TODO
 **Datastore**: Redis
 
 - Frameworks and libraries
+  - [`Redis`](https://redis.io/) is an open source NoSQL in-memory data store which can be used as a database and/or a cache
+- Considerations
+  - Pro: The [community](https://redis.io/) version of Redis is free
```

Comment:
```suggestion
```

Imo it should be a given that all the tools we choose here are free :) no proprietary tech plz

However, one thing that might be useful to point out is that there are a few options for free redis _deployments_:

* heroku (25mb) https://elements.heroku.com/addons/heroku-redis
* redis cloud (30mb) https://redislabs.com/redis-enterprise-cloud/pricing/

Reply:
tracking in https://github.com/ubclaunchpad/docs/issues/133